src/Sulu/Bundle/MediaBundle/Resources/js/containers/ImageMap/formRenderer.scss

```diff
@@ -1,12 +1,13 @@
 @import 'sulu-admin-bundle/containers/Application/colors.scss';
 
-$backgroundColor: $white;
-$borderColor: $silver;
-$labelColor: $scorpion;
+$formRendererBackgroundColor: $white;
+$formRendererBorderColor: $silver;
+$tabsBackgroundColor: transparent;
+$emptyTabsLabelColor: $scorpion;
```

Comment:
Also prefix the previous two variables for the same reason.

Reply:
fixed